packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,30 @@
-const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
-}
+// Routes constants
+const CREDITS_APPLICATION_BY_INDIVIDUAL_OR_ORGANISATION = 'credits-purchase/applying-individual-organisation'
```

Comment:
Don't need this, it's been removed from master branch anyway

Reply:
Its next route in this ticket